src/commands/synthetics/__tests__/reporters/junit.test.ts

```diff
@@ -0,0 +1,202 @@
+import {promises as fs} from 'fs'
+
+import {getDefaultStats, JUnitReporter} from '../../reporters/junit'
+import {RunTestCommand} from '../../run-test'
+
+const globalTestMock = {
+  locations: [],
+  options: {},
+  tags: [],
+}
+
+const globalResultMock = {
+  result: {stepDetails: [], device: {}},
+}
+
+describe('Junit reporter', () => {
+  const writeMock = jest.fn()
+  const commandMock: any = {
+    context: {
+      stdout: {
+        write: writeMock,
+      },
+    },
+    jUnitReport: 'junit',
+  }
+
+  // Using "any" to access private properties
+  let reporter: any
```

Comment:
Isn't there any other way possible?
In the worker code base, to keep the type that is being tested, we usually access private properties through the square-bracket notation and we let ESLint accept this notation: https://github.com/DataDog/synthetics-worker/blob/prod/worker/test/.eslintrc.js).

Reply:
Ah makes sense. Will update.

Reply:
Though, the typing is really getting in the way of testing if I don't use `any`.